src/fragments/lib/auth/android/sms/sign_in.mdx

```diff
@@ -0,0 +1,83 @@
+<BlockSwitcher>
+<Block name="Java">
+
+```java
+Amplify.Auth.signIn(
+    "username",
+    "password",
+    result -> {
+        if (result.getNextStep().getSignInStep() == AuthSignInStep.CONFIRM_SIGN_IN_WITH_SMS_MFA_CODE &&
+            result.getNextStep().getCodeDeliveryDetails() != null) {
+            String destination = result.getNextStep().getCodeDeliveryDetails().getDestination();
+            Log.d("SignIn", "SMS code sent to "+ destination);
+            Log.d("SignIn", "Additional Info" + result.getNextStep().getAdditionalInfo());
+
+            // Prompt the user to enter the SMSMFA code they received
+            // Then invoke `confirmSignIn` api with the code
```

Comment:
can we remove "with the code" here in all the code snippets in this block switcher.